Sources/Player/PlaybackEngine/Internal/Broadcasting/DJProducer.swift

```diff
@@ -34,6 +34,8 @@ final class DJProducer {
 	func start(_ title: String, with mediaProduct: MediaProduct, at position: Double) {
 		queue.dispatch {
 			guard self.curationUrl == nil else {
+				// ADD LOG
```

Comment:
Was 'Add log` a reminder/todo that we can now remove?

Reply:
Yes, I added as a note to myself before actually adding log. I will review it.

Reply:
removed ✅